framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
I think having a single way of defining "simba_jdbc" category is sufficient. Let's not use "smoke". Any test suite which needs to be added or updated for Simba JDBC should define the "simba_jdbc" category. I'm using it similarly for MapR-DB tests - they are categorized as "functional_maprdb".